website/docs/secrets/setup-secrets-operator.mdx

```diff
@@ -3,12 +3,125 @@ title: Setup Secrets Operator
+
+The profile consists of 2 components
+
+1- [The HelmChart for External Secrets Operator](https://github.com/weaveworks/profiles-catalog/blob/main/charts/external-secrets/Chart.yaml)  
+- This will install the operator helm chart and the operator CRDs. more info [here](https://external-secrets.io/v0.8.1/examples/gitops-using-fluxcd/)
+
+2- [Kustomization Reference to secret stores CRs](https://github.com/weaveworks/profiles-catalog/blob/main/charts/external-secrets/templates/secret-stores-kustomization.yaml)
+
+- This kustomization refers to the git repository with the secrets and can have the values from [values file](https://github.com/weaveworks/profiles-catalog/blob/main/charts/external-secrets/values.yaml) according to your secrets git repository
+
+### Installation Steps
+
+- [Bootstrap the required secrets](/secrets/bootstraping-secrets.mdx) for the Profile Helm Repository and the Secrets Repository
+- Install the [Profile Helm repository](https://docs.gitops.weave.works/docs/gitops-templates/cli/#profiles)
+- To install the operator on Management Cluster
+    - Go to `Applications` from the side bar and `Add new application`
+    - Select `External Secrets` and input your `namespace`
+        ![ESO Profile](./imgs/eso1.jpg)
+    - Then Adjust the `values` according to your secrets git repository
+        ![ESO Values](./imgs/eso2.jpg)
+    - After that Authenticate with Github, Create and merge the pull request
+        ![ESO PR](./imgs/eso3.jpg)
+    - Once Flux reconcile the git source this should take few minutes, you will find the operator installed and the secrets lives in the cluster
+- To bootstrap the operator on a leaf Cluster
+    - Go to `Clusters` from the side bar and `Create A Cluster`
+    - Select your template (must have the enable profiles annotation)
+        ![ESO Cluster](./imgs/eso5.jpg)
+    - Select `External Secrets` from and input your `namespace`
+        ![ESO Profile](./imgs/eso4.jpg)
+    - Then Adjust the `values` according to your secrets git repository
+        ![ESO Values](./imgs/eso2.jpg)
+    - After that Authenticate with Github, Create and merge the pull request
+        ![ESO PR](./imgs/eso3.jpg)
+    - Once the cluster is created and Flux reconcile the git source this should take few minutes, you will find the operator installed and the secrets lives in the cluster
+
 
 ## Set up SOPS
-// Explain how to set up SOPS on leaf clusters. 
\ No newline at end of file
+
+- Installing Sops on Managmenet cluster, follow [this guide](/secrets/setup-sops.mdx)
+
+### Bootstrapping Sops to leaf clusters
+
+import SopsBootstrapJob from "!!raw-loader!./assets/sops-bootstrap-job.yaml";
+import TemplateParams from "!!raw-loader!./assets/template-params.yaml";
+import TemplateAnnotations from "!!raw-loader!./assets/template-annotations.yaml";
+
+- **This will need the following**
+    - `ClusterBootstrapConfig` job to install sops and generate keys
```

Comment:
No need for this to be a bullet point, it ends creating to many levels of bullet points. You can add a small header if you want to highlight different sections, it will make things much more readable.